package.json

```diff
@@ -38,37 +38,37 @@
     "node": ">=6.11.5"
   },
   "dependencies": {
-    "@babel/cli": "^7.6.3",
-    "@babel/core": "^7.6.3",
-    "acorn": "^6.3.0",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.4",
+    "acorn": "^7.1.0",
     "babel-loader": "^8.0.6",
-    "chalk": "^2.4.2",
+    "chalk": "^3.0.0",
     "circular-dependency-plugin": "^5.2.0",
     "copy-template-dir": "1.4.0",
-    "cross-spawn": "6.0.5",
-    "eslint": "^6.5.1",
-    "inquirer": "^6.5.2",
-    "jest": "^24.9.0",
-    "ora": "^3.4.0",
-    "prettier": "^1.18.2",
-    "react-styleguidist": "^9.1.16",
-    "resolve": "^1.12.0",
+    "cross-spawn": "^7.0.1",
+    "eslint": "^6.8.0",
+    "inquirer": "^7.0.4",
+    "jest": "^25.1.0",
+    "ora": "^4.0.3",
+    "prettier": "^1.19.1",
+    "react-styleguidist": "^10.6.2",
+    "resolve": "^1.15.1",
     "resolve-pkg": "^2.0.0",
-    "rimraf": "^2.7.1",
+    "rimraf": "^3.0.2",
     "run-parallel": "1.1.9",
     "run-series": "1.1.8",
```

Comment:
Any particular reason `run-series` and `run-parallel` have exact versions instead of caret ranges?

Reply:
Huh, not sure why that is. I'll go ahead and update that.

Reply:
Ok, updated -- didn't result in any outdated deps.